src/shrutiC/OperationOnPrimeNumbers.java

```diff
@@ -0,0 +1,79 @@
+package shrutiC;
+
+public class OperationOnPrimeNumbers {
+
+	int primeNumSum;
+	int totalNumOfPrimeNos;
+
+	void finSumOfPrimeNumbers(int startNum, int endNum) {
+		int sum = 0;
+		for (int number = startNum; number <= endNum; number++) {
```

Comment:
when ever you write prime number code, make sure you are define range 2, reason we already seen in class